code/game/objects/items/weapons/implants/implant_death_alarm.dm

```diff
@@ -59,5 +50,20 @@
 /obj/item/implant/death_alarm/removed(mob/target)
 	if(..())
 		UnregisterSignal(target, COMSIG_MOB_DEATH)
-		return 1
-	return 0
+		return TRUE
+	return FALSE
+
+/obj/item/implanter/death_alarm
+	name = "implanter (Death Alarm)"
+
+/obj/item/implanter/death_alarm/Initialize(mapload)
+	imp = new /obj/item/implant/death_alarm(src)
+	..()
+
+/obj/item/implantcase/death_alarm
+	name = "Glass Case- 'Death Alarm'"
```

Comment:
Any reason this one's a "Glass Case" while the abductor one is an "Implant Case"?

Also, spacing on the hyphen.

Reply:
erghh, old code. I'll double check all of them for consistency and ensure they're all implant case

Reply:
adressed